src/main/resources/api/swagger.yaml

```diff
@@ -222,6 +218,15 @@ definitions:
             $ref: '#/definitions/Link'
           next:
             $ref: '#/definitions/Link'
+  Review:
+    type: object
+    properties:
+      review-status:
```

Comment:
Why not just `status`? For instance, we don't have `feature-name`, `feature-author`, etc. on the other entities. :)